src/ring_theory/norm.lean

```diff
@@ -195,7 +195,8 @@ begin
   letI := classical.dec_eq E,
   rw [norm_gen_eq_prod_roots pb hE, fintype.prod_equiv pb.lift_equiv', finset.prod_mem_multiset,
     finset.prod_eq_multiset_prod, multiset.to_finset_val,
-    multiset.erase_dup_eq_self.mpr (nodup_roots ((separable_map _).mpr hfx)), multiset.map_id],
+    multiset.erase_dup_eq_self.mpr, multiset.map_id],
+  { exact nodup_roots ((separable_map _).mpr hfx) },
```

Comment:
i'm not sure why this is necessary, `nodup_roots` only changed from `field K` to `comm_ring R`, `is_domain R`!